import-export-cli/impl/getApis.go

```diff
@@ -102,6 +102,7 @@ func (a *api) MarshalJSON() ([]byte, error) {
 // @return error
 func GetAPIListFromEnv(accessToken, environment, query, limit string) (count int32, apis []utils.API, err error) {
 	apiListEndpoint := utils.GetApiListEndpointOfEnv(environment, utils.MainConfigFilePath)
+	fmt.Println("API List Endpoint: ", apiListEndpoint)
```

Comment:
Do we need this?

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)